integration/test-mb/test-amqp/pom.xml

```diff
@@ -127,6 +127,7 @@
         <dependency>
             <groupId>org.wso2.andes.wso2</groupId>
             <artifactId>andes-client</artifactId>
+            <version>${andes.version}</version>
```

Comment:
Ideally the version should be inherited from the parent (product-ei) pom.xml if you add this dependency there.